content/tracing/getting_further/metrics_namespace.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Trace Metrics Namespace
+kind: documentation
+further_reading:
+- link: "/tracing/setup/"
+  tag: "Documentation"
+  text: Learn how to setup APM tracing with your application
+- link: "/tracing/visualization/services_list/"
+  tag: "Documentation"
+  text: Discover the list of services reporting to Datadog
+- link: "/tracing/visualization/service"
+  tag: "Documentation"
+  text: Learn more about services in Datadog
+- link: "/tracing/visualization/resource"
+  tag: "Documentation"
+  text: Dive into your resource performances and traces
+- link: "/tracing/visualization/trace"
+  tag: "Documentation"
+  text: Understand how to read a Datadog Trace
+---
+
+The trace metrics namespace is `trace.<name>.<metrics>{<tags>}` where
+
+* `<name>`: is the name of the operation i.e. the `span.name`: (*redis.command*, *pylons.request*, *rails.request*, *mysql.query*)
+* `<metrics>` is about the hits, errors or latency ("request.hits", etc.)
+* `<tags>` the metrics are tagged by service & resource.
```

Comment:
There are no periods at the end of the first two bullet points, so I would either remove this period, or add periods to the proceeding points.

Reply:
👌